v4m-container/Dockerfile

```diff
@@ -25,9 +25,12 @@ ENV KEEP_TMP_DIR=true
 RUN mkdir -p $V4M_PATH/bin $V4M_PATH/.kube $V4M_PATH/user_dir
 WORKDIR $V4M_PATH
 
+# Create $V4M_PATH/.kube/config if it doesn't exist
+RUN touch ${KUBECONFIG}
```

Comment:
Did you test this change for the case where the config file does exist? 
I'm just wondering if this command creating it, causes issue on line 32 when trying to copy that file over or does it just overwrite them empty version created here?

Reply:
I tested the change locally.

If the file doesn't exist, it creates the file and assigns the empty file those permissions.
If the file does exist, the touch updates the "Last Modified" date.